Assignment1

```diff
@@ -0,0 +1,30 @@
+#include<iostream>
+using namespace std;
+int Array_Size;
+__global__ void Sum(int* d1_in, int* d2_in, int* d_out,int* d_array_size)
+{
+    int i = blockIdx.x * blockDim.x + threadIdx.x;
+    if(i<*d_array_size)
+        d_out[i] = d1_in[i] + d2_in[i];
+}
+int main()
+{
+    int h1_in[Array_Size], h_out[Array_Size],h2_in[Array_Size];
+    int Array_Bytes = Array_Size * sizeof(int);  
```

Comment:
Hi! You didn't write the code for setting h1_in and h2_in. You can set them randomly (using rand()) or h1_in[i] = i, h2_in[i] = 2*i (whatever logic you want). Also, Array_Size being a global variable is 0. The aim is to use GPU for large number of inputs.